src/batch_group/repo.rs

```diff
@@ -95,7 +95,32 @@ impl BatchGroups {
         }
         Ok(BatchGroup::try_from(events)?)
     }
-
+    pub async fn list_by_account_id(
+        &self,
+        account_id: AccountId,
+    ) -> Result<Vec<BatchGroup>, BriaError> {
+        let rows = sqlx::query!(
+            r#"
+              SELECT b.*, e.sequence, e.event
+              FROM bria_batch_groups b
+              JOIN bria_batch_group_events e ON b.id = e.id
+              WHERE account_id = $1
+              ORDER BY b.id, e.sequence"#,
+            Uuid::from(account_id),
```

Comment:
this syntax is outdated. I haven't updated it everywhere but the way to do it is:
`account_id as AccountId` instead of the `Uuid::from` - any place you see `Uuid::from` in repo's you can updated it if you are working on the file.

Reply:
Also anytime you edit or add a sqlx query you will need to run `cargo sqlx prepare` to update the sqlx-data.json cache or `make check-code` won't pass.